src/views/Projects/index.jsx

```diff
@@ -20,11 +21,32 @@ import ProjectCard from '../../components/ProjectCard';
 export default class Projects extends Component {
   state = {
     projects,
```

Comment:
`projects` doesn't need to be in state because you are not doing `this.setState({ projects: ... })` anywhere in the component.